dev/.rat-excludes

```diff
@@ -97,4 +97,4 @@ org.apache.spark.scheduler.SparkHistoryListenerFactory
 LZ4BlockInputStream.java
 spark-deps-.*
 .*csv
-.*tsv
+.*tsv
```

Comment:
Any reason we removed the newline at the end here? I think we generally end files with a newline.